tests/filecheck/mlir-conversion/with-mlir/mlir_opt.mlir

```diff
@@ -1,4 +1,5 @@
 // RUN: xdsl-opt %s -p mlir-opt{arguments='--cse','--mlir-print-op-generic'} --print-op-generic | filecheck %s
+// RUN: xdsl-opt %s -p mlir[cse] --print-op-generic | filecheck %s
```

Comment:
This will run both tests, right? I think it's good to run and test this mlir-shorthand on the CI, but I think running twice here does not make sense. I would prefer to delete the upper line since the short-hand is now our way of interacting with mlir. What do you guys think?

Reply:
I wouldn't overwrite the first line, as my addition is more like sugar on top of it than intended as a replacement (it does not have the full flexibility of the underlying pass).
I get that this part is confusing; it's just me being lazy to be fair; I could split it in two filechecks; would that make sense to you?

Reply:
Oh, or you meant that it's redundant to test and the sugar and the underlying pass in a filecheck? I can agree with that and remove the second line; the sugar syntax is tested in the pipeline parser tests already.

Reply:
My point is that people will be looking at the tests to figure out how to use certain infrastructure. I think it makes more sense here to only have the new syntax because this is how we want people to interact with the infrastructure.
And given that both lines test the exact same thing (plus the conversion that is tested somewhere else), I think it does not matter which we choose from a testing perspective. So, I would go with the "user will look for examples" argument and use the latter.
This is also really only a minor comment, I don't mind running twice or having either of the two lines. This is just my feeling of what I think would be the best of all the choices :)

Reply:
Ah, I see;
I would still keep an example with the underlying pass, as it is still expected to be exposed. But I can switch that case to use it in a way that the `mlir[]` does not allow and add comments!